### ISSUE TITLE
Version checking - IndexError: list index out of range

### ISSUE THREAD

**Issue by FedericoComoglio** [carjed/helmsman#15]
Dear developers,

I downloaded the latest version of helmsman, using `conda` (recommended procedure). However, a simple test call currently raises an `IndexError` (see below). 

I figured that `version = files[-1]` raises and exception because `files` is an empty list. Commenting out all lines that refer to `version` works. Could you please look into this?

Thanks a lot!
Federico

```
Traceback (most recent call last):
  File "helmsman_debug.py", line 410, in <module>
    main()
  File "helmsman_debug.py", line 33, in main
    version = files[-1]
IndexError: list index out of range
(helmsman) ________________________________________________________________________________
```

**Comment by gprashant17** [comment]
I am facing the same issue. What files are supposed to be present in that directory? Also, when I try to comment the lines which refer to `version` and run, I get the following:

```
[helmsman::main] INFO ----------------------------------
[helmsman::main] INFO ----------------------------------
[helmsman::main] INFO random seed: 4730
[helmsman::main] INFO Total runtime: 20.48 seconds
```
No output file is generated in the specified directory.

**Comment by FedericoComoglio** [comment]
@gprashant123 - I confirm that no output is written to the output directory.

**Comment by carjed** [comment]
Thanks for bringing this up—I previously made a change to the version checking procedure in #11 to allow Helmsman to be used as a submodule. The new procedure worked for me locally, but it seems to have broken things when pulled from the repo. For a quick fix, I've just hard-coded the version number, but will try to implement something a bit more elegant for my own sanity :)

As for the missing output files, I realized it would only output if the `--decomp` option was used. Fixed this in #16, so now if Helmsan is run without `--decomp`, it will still output the count & normalized spectra matrices to the specified output directory.
